web/src/pages/login.tsx

```diff
@@ -0,0 +1,36 @@
+import { LoginEmailCheck } from "@/components/LoginEmailCheck";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { getActiveUser } from "@/lib/auth/sessionHelper";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { useMountEffect } from "@/lib/utils/helpers";
+import { type GetStaticPropsResult } from "next";
+import { useRouter } from "next/router";
+
+import { ReactElement } from "react";
+
+const LoginEmailCheckPage = (): ReactElement => {
+  const router = useRouter();
+
+  useMountEffect(() => {
+    getActiveUser()
+      .then(() => {
+        router.push(ROUTES.dashboard);
+      })
+      .catch(() => {});
+  });
```

Comment:
I'm mimicking [this pattern from _app.ts](https://github.com/newjersey/navigator.business.nj.gov/blob/main/web/src/pages/_app.tsx#L110-L123), but with slightly different behavior:

- If someone is logged in and somehow navigates to this page, just redirect them to the dashboard
- otherwise, render the login page

[the value return from getStaticProps below](https://github.com/newjersey/navigator.business.nj.gov/pull/9276/files#diff-1660c6bd43af043e9efbae24c6874f5296f839c627e9e0cf75fc06446a328738R30-R34) just prevents some redirects that happen in `_app.tsx`